### DISCUSSION TITLE
htmlTable Formatierungen

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1375] [Q&A]
Sollten beim htmlTable nicht auch die Formatierungen (HTML-CSS) mit übernommen werden in die Tabelle?
font-size, color, background.color usw.?
![image](https://user-images.githubusercontent.com/84818934/147413466-7190e7ed-2b5a-4d4e-b194-90987d66302f.png)
![image](https://user-images.githubusercontent.com/84818934/147413783-d50515d8-0130-428e-a2e4-f1547c921e8a.png)

**Comment by Zefau** [comment]
Aktuell nicht. Irgendwann ja. Bin mir unsicher ob es schon einen Feature Request gibt, steht aber auf der Agenda
